pkgs/applications/video/vokoscreen-ng/linux-support-installation-target.patch

```diff
@@ -1,9 +1,8 @@
-Seulement dans b: patch.patch
```

Comment:
can we fetchpatch the arch patch https://github.com/archlinux/svntogit-community/blob/packages/vokoscreen/trunk/0001-Add-better-integration-for-Linux.patch